### PR TITLE
feat(health-sdk): Include `MoreInformation` in `BankSelectionBottomSh…

### PR DESCRIPTION
**Pull request by danicretu**
…eet`

IPC-248

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
Thank you, looks great!

File: health-sdk/sdk/src/main/java/net/gini/android/health/sdk/util/extensions/Context.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal fun Context.generateBitmapDrawableIcon(icon: ByteArray, iconSize: Int):
                 iconSizePx,
                 true
             )
-            bitmap.recycle()
+            if (bitmap.width != iconSizePx) bitmap.recycle()
```
Comment (a-szotyori) [inline health-sdk/sdk/src/main/java/net/gini/android/health/sdk/util/extensions/Context.kt:27]: [Suggestion] Could you add please a short comment explaining this fix? It's not obvious why it's needed and would help us understand in the future why it was added. 🙏

Comment (danicretu) [reply health-sdk/sdk/src/main/java/net/gini/android/health/sdk/util/extensions/Context.kt:30]: Added a short explanation, hopefully it makes sense.

Comment (a-szotyori) [reply health-sdk/sdk/src/main/java/net/gini/android/health/sdk/util/extensions/Context.kt:30]: Thank you, it makes sense 👍 😄

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**